src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -445,6 +469,11 @@ protected void generate0()
             totalConferencesCompleted
                 += jvbStats.totalConferencesCompleted.get();
             totalConferenceSeconds += jvbStats.totalConferenceSeconds.get();
+            totalParticipantSeconds += jvbStats.totalParticipantSeconds.get();
```

Comment:
this is confusing to me.  we use `+=` but isn't this really just trying to achieve `totalParticipantSeconds = jvbStats.totalParticipantSeconds.get()`?  we don't actually accumulate multiple values in `totalParticipantSeconds`, right?  it should just be set to the value of `jvbStats.totalParticipantSeconds`?

Reply:
In theory there can be multiple Videobridge instances, so totalX is an accumulator.